soweego/validator/checks.py

```diff
@@ -109,12 +118,27 @@ def dead_ids_cli(catalog, entity, deprecate, sandbox, dump_wikidata, dir_io):
             fout,
             indent=2,
         )
-
     LOGGER.info('Dead identifiers dumped to %s', dead_ids_path)
 
+    # Dump Wikidata cache
+    if dump_wikidata:
```

Comment:
this part is also repeated a few times, maybe worth to make a class for the cache so it's also easier to switch to a new backend if needed

Reply:
Great idea, filed in #421